charts/zigbee2mqtt/README.md

```diff
@@ -0,0 +1,66 @@
+# zigbee2mqtt: Fully configurable Zigbee to MQTT Gateway
+
+This is a helm chart for [zigbee2mqtt](https://www.zigbee2mqtt.io)
+
+## TL;DR;
+
+```shell
+$ helm repo add billimek https://billimek.com/billimek-charts/
+$ helm install billimek/zigbee2mqtt
+```
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+helm install --name my-release billimek/zwave2mqtt
```

Comment:
Can you please change this to `zigbee2mqtt` ?

Reply:
Oh my, looks like atom didn't save my version of this file. Sorry about that